### PR TITLE
Add KCC 2022 contest article

### PR DESCRIPTION
**Pull request by Garalulu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Walavouchey** [comment]
looks like this is still a wip (missing banner, song list, results sheet). you don't need to include things like the results sheet, song list or badges if they don't exist yet.

**Comment by Garalulu** [comment]
> looks like this is still a wip (missing banner, song list, results sheet). you don't need to include things like the results sheet, song list or badges if they don't exist yet.

fixed!

File: wiki/Contests/KCC/2/en.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+tags:
+  - KCC2022
+  - KCC 2022
+---
+
+# Korean Charting Contest 2022
+
+![KCC2022 banner](img/banner.png)
+
+The **Korean Charting Contest 2022** (***KCC2022***) is a osu!mania mapping contest for users from ![][flag_KR] South Korea.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:11]:
```suggestion
The **Korean Charting Contest 2022** (***KCC2022***) is an osu!mania mapping contest for users from ![][flag_KR] South Korea.
```

File: wiki/Contests/KCC/2/en.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Contest schedule
+
+| Event | Timestamp (UTC+09) |
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:15]:
```suggestion
| Event | Timestamp (UTC+9) |
```

File: wiki/Contests/KCC/2/en.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1507792)
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!mania mode only.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:48]:
```suggestion
- This contest is for the osu!mania mode only.
```

File: wiki/Contests/KCC/2/en.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+- Only Koreans can participate in this contest. If your profile flag is not showing that you're in ![][flag_KR] South Korea for reasons like foreign residence, you can still participate as long as you're able to verify that you are Korean.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:49]:
```suggestion
- Only Koreans can participate in this contest. If your profile flag does not show that you are in ![][flag_KR] South Korea for reasons such as foreign residence, you can still participate as long as you are able to verify that you are Korean.
```

File: wiki/Contests/KCC/2/en.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+- You can map one of total six songs, three songs each from Instrumental and Vocal groups.
+- The osu! client must be able to load and read your submission.
+- Submission must be in `.osz` format.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:52]:
```suggestion
- Submissions must be in the `.osz` format.
```

File: wiki/Contests/KCC/2/en.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+- Custom hitsounds are allowed.
+- Title, Artist, MP3, Source, Tags and Background must NOT be modified. However, preview point can be customised.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:54]:
```suggestion
- Title, Artist, MP3, Source, Tags, and Background must NOT be modified. However, the preview point can be customised.
```

File: wiki/Contests/KCC/2/en.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+- Videos, skins and storyboards are NOT allowed. However, storyboard hitsounds can be modified.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:55]: there are no storyboarded hitsounds by default, so there's nothing to modify

```suggestion
- Videos, skins, and storyboards are NOT allowed. However, storyboarded hitsounds are allowed.
```

File: wiki/Contests/KCC/2/en.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+- Submission must be 4K ~ 9K.
+- The Auto mod must be able to full combo (1,000,000 score) your submission.
+- Submission must be finished by the contestant alone. No collaboration is allowed.
+- Submission must not be uploaded to the osu! website through the Beatmap Submission System (BSS). All contest submissions are anonymous during the judging phases to avoid bias.
+- Only one beatmap difficulty is allowed. If multiple difficulties are submitted, then the one with the highest star difficulty will be accepted.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:60]:
```suggestion
- Only one beatmap difficulty is allowed. If multiple difficulties are submitted, then the one with the highest star rating will be accepted.
```

File: wiki/Contests/KCC/2/en.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+- Only the following difficulties are allowed: Insane or higher. Please check the difficulty-specific rules in the [ranking criteria](/wiki/Ranking_Criteria).
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:61]:
```suggestion
- Only Insane or higher difficulties are allowed. Please check the [difficulty-specific rules in the ranking criteria](/wiki/Ranking_Criteria/osu!mania#difficulty-specific).
```

File: wiki/Contests/KCC/2/en.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+- If you submit several maps, the last one will be your final entry.
+- Submitting entries means understanding and agreement to the ruleset above.
+
+### Judging criteria
+
+For each eligible submitted difficulty, the judging panel will award points based on the following criteria:
+
+- **Expertise (25 pts):** How well the submission demonstrates techniques regarding structure, music representation, flow, hitsounding, etc.
+- **Cohesion (25 pts):** How consistent the submission is regarding structure, music representation, flow, hitsounding etc especially between different sections.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:70]:
```suggestion
- **Cohesion (25 pts):** How consistent the submission is regarding structure, music representation, flow, hitsounding, etc., especially between different sections.
```

File: wiki/Contests/KCC/2/en.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+- **Creativity (25 pts):** How well the submission shows originality and uniqueness compared with the other submissions with reasonable respect to the playability.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:71]:
```suggestion
- **Creativity (25 pts):** How well the submission shows originality and uniqueness compared to the other submissions, with reasonable consideration for playability.
```

File: wiki/Contests/KCC/2/en.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+- **Judges' Impression (25 pts):** This is the part of the score left to the Judges' personal preferences on the submission.
+
+[flag_CN]: /wiki/shared/flag/CN.gif "China"
+[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
+[flag_SG]: /wiki/shared/flag/SG.gif "Singapore"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:77]: usually sorted by country code and not country names

```suggestion
[flag_CN]: /wiki/shared/flag/CN.gif "China"
[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
[flag_SG]: /wiki/shared/flag/SG.gif "Singapore"
```

File: wiki/Contests/KCC/2/ko.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+
+- **Expertise (25 pts):** 맵이 구조, 노래 표현, 흐름과 관련된 기술력을 얼마나 잘 보여주고 있는가
+- **Cohesion (25 pts):** 맵이 구조, 음악 표현, 흐름에 대해 얼마나 일관성을 가지고 있는가
+- **Creativity (25 pts):** 다른 맵과 비교했을 때 플레이성 면에서 독창성과 특별함을 가지고 있는가
+- **Judge's Impression (25 pts):** 심사위원들의 개인적 선호도를 나타내는 점수
+
+[flag_CN]: /wiki/shared/flag/CN.gif "중국"
+[flag_KR]: /wiki/shared/flag/KR.gif "대한민국"
+[flag_SG]: /wiki/shared/flag/SG.gif "싱가포르"
+[flag_PH]: /wiki/shared/flag/PH.gif "필리핀"
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/ko.md:77]:
```suggestion
[flag_CN]: /wiki/shared/flag/CN.gif "중국"
[flag_KR]: /wiki/shared/flag/KR.gif "대한민국"
[flag_PH]: /wiki/shared/flag/PH.gif "필리핀"
[flag_SG]: /wiki/shared/flag/SG.gif "싱가포르"
```

**Comment by Garalulu** [comment]
fixed all (did for kcc2021 cuz of consistency) + added song list

**Comment by Garalulu** [comment]
> fixed all (did for kcc2021 cuz of consistency) + added song list

@Walavouchey

**Comment by ZeroPyrozen** [comment]
You may do self-check and if it all has completed then check the list above. Also resolve above suggestions.

File: wiki/Contests/KCC/2/ko.md [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+- Tokyo Jihen - Sounan (3:20) / 156bpm
+- Kabocha feat. Sennzai - Morphobia (3:54) / 145bpm
+
+## 규칙
+
+### 대회 규칙
+
+- osu!mania 모드 전용 대회입니다.
+- '한국인'만 참가 가능합니다. 외국 거주, 교포 2세 등의 사유로 osu! 내에서의 국기가 대한민국이 아닌 경우, 이미 한국인이라고 널리 알려져 있거나 어떠한 방식으로든 입증만 가능하다면 한국인으로서 참가할 수 있습니다.
+- 참가자들은 사전에 선정되어진 Instrumental과 Vocal 그룹 각각 3곡 총 6곡 중 하나를 매핑해 제출할 수 있습니다.
```
Comment (libneko) [inline wiki/Contests/KCC/2/ko.md:64]: 한국어에서는 굳이 저걸 "선정되어진"이라고 수동으로 적을 필요는 없습니다. 오히려 "선정된"이 더 자연스러울 수 있겠네요.

**Review by libneko — COMMENTED** [review]
번역된 문서쪽에서는 크게 신경쓸건 없네요.

File: wiki/Contests/KCC/1/en.md [hunk 15]
```diff
@@ -8,11 +8,11 @@ tags:
 
 ![KCC2021 banner](img/banner.png)
 
-The **Korean Charting Contest 2021** (***KCC2021***) is a osu!mania mapping contest for users from ![][flag_KR] South Korea. It was the first instalment of the Korean Charting Contest.
+The **Korean Charting Contest 2021** (***KCC2021***) is an osu!mania mapping contest for users from ![][flag_KR] South Korea. It was the first installment of the Korean Charting Contest.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/1/en.md:11]: the wiki uses british spelling

```suggestion
The **Korean Charting Contest 2021** (***KCC2021***) is an osu!mania mapping contest for users from ![][flag_KR] South Korea. It was the first instalment of the Korean Charting Contest.
```

File: wiki/Contests/KCC/1/en.md [hunk 16]
```diff
@@ -78,20 +78,20 @@ The **Korean Charting Contest 2021** (***KCC2021***) is a osu!mania mapping cont
...
+- This contest is for the osu!mania mode only.
+- Only Koreans can participate in this contest. If your profile flag does not show that you are in ![][flag_KR] South Korea for reasons such as foreign residence, you can still participate as long as you are able to verify that you are Korean.
 - You should pick a song that you want to map when you register (one song for Instrumental / Vocal each). You can map one of those two songs after the voting ends.
 - The osu! client must be able to load and read your submission.
-- Submission must be in `.osz` format.
+- Submissions must be in the `.osz` format.
 - Custom hitsounds are allowed.
-- Title, Artist, MP3, Source, Tags and Background must NOT be modified. However, preview point can be customised.
-- Videos, skins and storyboards are NOT allowed. However, storyboard hitsounds can be modified.
+- Title, Artist, MP3, Source, Tags and Background must NOT be modified. However, the preview point can be customized.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/1/en.md:87]:
```suggestion
- Title, Artist, MP3, Source, Tags and Background must NOT be modified. However, the preview point can be customised.
```

File: wiki/Contests/KCC/2/en.md [hunk 17]
```diff
@@ -0,0 +1,91 @@
...
+tags:
+  - KCC2022
+  - KCC 2022
+---
+
+# Korean Charting Contest 2022
+
+![KCC2022 banner](img/banner.png)
+
+The **Korean Charting Contest 2022** (***KCC2022***) is an osu!mania mapping contest for users from ![][flag_KR] South Korea.
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:11]:
```suggestion
The **Korean Charting Contest 2022** (***KCC2022***) is an osu!mania mapping contest for users from ![][flag_KR] South Korea. It is the second instalment of the Korean Charting Contest.
```

File: wiki/Contests/KCC/2/en.md [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1507792)
+
+## Song list
+
+### Instrumental
+
+- lapix - shime (Extended Mix) (3:44) / 160bpm
+- Mitsukiyo - rolling beat (2:31) / 170bpm
+- Laur - PRAGMATISM -RESURRECTION- (2:36) / 174bpm
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:50]:
```suggestion
| Song | Length | BPM |
| :-- | :-- | :-- |
| lapix - shime (Extended Mix) | 3:44 | 160 |
| Mitsukiyo - rolling beat | 2:31 | 170 |
| Laur - PRAGMATISM -RESURRECTION- | 2:36 | 174 |
```

File: wiki/Contests/KCC/2/en.md [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+
+### Vocal
+
+- fishbowl feat.Yukichi Kasaku/men - Shinkai (3:37) / 133bpm
+- Tokyo Jihen - Sounan (3:20) / 156bpm
+- Kabocha feat. Sennzai - Morphobia (3:54) / 145bpm
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/en.md:56]:
```suggestion
| Song | Length | BPM |
| :-- | :-- | :-- |
| fishbowl feat.Yukichi Kasaku/men - Shinkai | 3:37 | 133 |
| Tokyo Jihen - Sounan | 3:20 | 156 |
| Kabocha feat. Sennzai - Morphobia | 3:54 | 145 |
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
it seems like this pr was created without first updating the base branch, which didn't include some changes to the 2021 article. please follow [this step in the contribution guidelines](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#syncing-the-fork) in the future

also you don't need to merge the master branch yourself - a maintainer will do this right before the pr is merged

File: wiki/Contests/KCC/2/ko.md [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
+
+## 링크
+
+- [포럼 스레드](https://osu.ppy.sh/community/forums/topics/1507792)
+
+## 곡 리스트
+
+### Instrumental
+
+- lapix - shime (Extended Mix) (3:44) / 160bpm
```
Comment (libneko) [inline wiki/Contests/KCC/2/ko.md:48]: 위 내용 적용하신다면 이것들도 같이 수정해주시긴 해야합니다.

```
| 곡 | 길이 | BPM |
| :-- | :-- | :-- |
| lapix - shime (Extended Mix) | 3:44 | 160 |
| Mitsukiyo - rolling beat | 2:31 | 170 |
| Laur - PRAGMATISM -RESURRECTION- | 2:36 | 174 |
```

File: wiki/Contests/KCC/2/ko.md [hunk 21]
```diff
@@ -0,0 +1,91 @@
...
+- Mitsukiyo - rolling beat (2:31) / 170bpm
+- Laur - PRAGMATISM -RESURRECTION- (2:36) / 174bpm
+
+### Vocal
+
+- fishbowl feat.Yukichi Kasaku/men - Shinkai (3:37) / 133bpm
```
Comment (libneko) [inline wiki/Contests/KCC/2/ko.md:54]: 마찬가지로 그대로 적용할거라면 표로 만들어주시긴 해야합니다

```
| 곡 | 길이 | BPM |
| :-- | :-- | :-- |
| fishbowl feat.Yukichi Kasaku/men - Shinkai | 3:37 | 133 |
| Tokyo Jihen - Sounan | 3:20 | 156 |
| Kabocha feat. Sennzai - Morphobia | 3:54 | 145 |
```

**Review by libneko — CHANGES_REQUESTED** [review]
wiki/Contents/KCC/1/en.md와 wiki/Contests/KCC/1/ko.md도 같이 수정을 하면 좋을 것 같긴 하네요. 따로 이야기 없다 싶으면 그냥 놔두셔도 되긴 할거라 생각됩니다.

File: wiki/Contests/KCC/2/ko.md [hunk 22]
```diff
@@ -0,0 +1,91 @@
...
+tags:
+  - KCC2022
+  - KCC 2022
+---
+
+# Korean Charting Contest 2022
+
+![KCC2022 배너](img/banner.png)
+
+**Korean Charting Contest 2022** (***KCC2022***) 은 한국인을 위한 osu!mania 매핑 대회입니다.
```
Comment (libneko) [inline wiki/Contests/KCC/2/ko.md:11]: 이전 문서에도 있던 내용이라 "이 대회는 두 번째 대회입니다"도 추가하는게 맞다고 봅니다. 위키 관리자분이 Request changes에서 언급한 내용이기도 하고요.

**Review by libneko — CHANGES_REQUESTED** [review]

**Comment by Garalulu** [comment]
all fixed, sorry for the disturbance!

File: wiki/Contests/KCC/2/ko.md [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+
+## 링크
+
+- [포럼 스레드](https://osu.ppy.sh/community/forums/topics/1507792)
+
+## 곡 리스트
+
+### Instrumental
+
+| Song | Length | BPM |
```
Comment (Walavouchey) [inline wiki/Contests/KCC/2/ko.md:48]: shouldn't these be translated? maybe you accidentally copied the english version instead of libneko's review

Comment (Garalulu) [reply wiki/Contests/KCC/2/ko.md]: at first I don't think it should be translated too but after talking with libneko, I changed my mind. fixed!

**Review by libneko — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**